wallet/src/Ergvein/Wallet/Storage/Keys.hs

```diff
@@ -80,6 +82,14 @@ derivePubKey masterKey keyPurpose index =
     ErgXPubKey k _ -> ErgXPubKey (derivedKey k) ""
     BtcXPubKey k _ -> BtcXPubKey (derivedKey k) ""
 
+-- | Extract addresses from keystore
+extractAddrs :: PubKeystore -> [(Maybe Int, EgvAddress)]
+extractAddrs (PubKeystore mast ext int) = mastadr:(extadrs <> intadrs)
```

Comment:
We never use address derived from master pubkey. And the function should reuse already existed function for getting all public keys https://github.com/hexresearch/ergvein/blob/3214aded1aed6de3a432c192bf0b84a3133f7273/wallet-types/src/Ergvein/Types/Keys.hs#L351. Note that if we don't need master key, we don't need `Maybe` part for index and that external and public keys have independent indexing.